### ISSUE TITLE
Miss-formatting LESS file

### ISSUE THREAD

**Issue by edoher** [beautify-web/js-beautify#745]
Hi, the plugin so far works great. Today I found an issue that took me a lot of effort to understand.

```less
#main > header.big-menu nav ul.level-3 > li > ul li div a:hover {
    &:extend(#main > header.big-menu nav ul li > a: hover);
}
```
Beautify added a random space in that piece of code, within a LESS file of over 1000 lines, making it impossible for me to compile it and even harder for me to understand what was wrong.

For some reason, Beautify added a space in "a:hover", making it be "a: hover", therefore rendering it invalid.

Please check that error, because probably I'm not the only one who had a hard time with it.

Thanks!

**Comment by bitwiseman** [comment]
You are not the only one.
I think this is a duplicate of #618.

**Comment by bitwiseman** [comment]
@edoher  I believe this is fixed in 1.6.4.  Please reopen if it is still present.
